conda/plugins/subcommands/doctor/health_checks.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+from __future__ import annotations
+
+import json
+
+from pathlib import Path
+
+from conda.base.context import context
+
+OK_MARK = "✅"
+REPORT_TITLE = "\nENVIRONMENT HEALTH REPORT\n"
+DETAILED_REPORT_TITLE = "\nDETAILED ENVIRONMENT HEALTH REPORT\n"
+MISSING_FILES_SUCCESS_MESSAGE = f"{OK_MARK} There are no packages with missing files.\n"
+
+
+def display_report_heading() -> None:
```

Comment:
This would be easier to test if it took name as an argument (avoid accessing context as much as possible, possibly only pass context in from the cli module and have all other functions take paths)